komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -665,7 +665,7 @@ export default Vue.extend({
       if (this.single && this.poster.uploadQueue.length > 0) {
         const series = this.series as SeriesDto
         let hadErrors = false
-        for (const file of this.poster.uploadQueue) {
+        for (const file of this.poster.uploadQueue.slice()) {
```

Comment:
what would that do exactly?

Reply:
not sure what would be the best way to do that, but here we iterate over  poster.uploadQueue array and delete it's elements in the process

So instead of iterating over original array we create a copy